concrete/controllers/single_page/dashboard/system/basics/multilingual.php

```diff
@@ -16,6 +19,18 @@ public function view()
         $this->set('LANGUAGE_CHOOSE_ON_LOGIN', Config::get('concrete.i18n.choose_language_login'));
         $this->set('interfacelocales', $locales);
         $this->set('SITE_LOCALE', Config::get('concrete.locale'));
+        $mlLink = null;
+        $mlPage = Page::getByPath('/dashboard/system/multilingual/setup');
+        if ($mlPage && !$mlPage->isError()) {
+            $cp = new Checker($mlPage);
+            if ($cp->canViewPage()) {
+                $mlLink = [
+                    t($mlPage->getCollectionName()),
+                    $this->app->make(ResolverManagerInterface::class)->resolve([$mlPage]),
```

Comment:
Well, I usually avoid using `$page->getCollectionLink()` because it has a lot of overhead:

- `$page->getCollectionLink()` calls `Core::make('helper/navigation')->getLinkToCollection()`
- `Core::make('helper/navigation')` implies `__callStatic` and its logic
- `Concrete\Core\Html\Service\Navigation::getLinkToCollection()` calls `URL::to`
- `URL::to` implies `__callStatic` and its logic

BTW, in this PR we have just one-time calls, so yes, let's use `$page->getCollectionLink()` (see 333cc9b)